src/zenml/cli/served_models.py

```diff
@@ -16,20 +16,23 @@
 
 import click
 
-from zenml.cli.cli import cli
+from zenml.cli.cli import TagGroup, cli
 from zenml.cli.utils import (
     declare,
     error,
     pretty_print_model_deployer,
     print_served_model_configuration,
     warning,
 )
-from zenml.enums import StackComponentType
+from zenml.enums import CliCategories, StackComponentType
 from zenml.model_deployers import BaseModelDeployer
 from zenml.repository import Repository
 
 
-@cli.group()
+@cli.group(
+    cls=TagGroup,
+    tag=CliCategories.MODEL_DEPLOYMENT,
+)
 @click.pass_context
 def served_models(ctx: click.Context) -> None:
     """List and manage served models that are deployed using the active model
```

Comment:
Can we perhaps make this help message a bit shorter to make all lines on the main CLI page 1-liners?

Reply:
I tried to make it shorter, not sure if I could come up with a shorter message for these two help messages : 
-       served-models           "List and manage served models with the active model deployer."
-       integration             "Interact with the requirements of external integrations."
otherwise, I think it does look okay for these two commands since they are in individual tags